src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final int dataDate = 0;
+        final int dataName = 1;
+        final int dataHours = 2;
+        final int dataIncome = 3;
+        LocalDate salaryDateFrom = LocalDate.parse(dateFrom, formatter);
+        LocalDate salaryDateTo = LocalDate.parse(dateTo, formatter);
+        StringBuilder builder = new StringBuilder();
```

Comment:
we need to specify what this builder is for. 
```suggestion
        StringBuilder reportBuilder = new StringBuilder();
```